x-pack/plugins/security_solution/public/management/common/utils.ts

```diff
@@ -19,3 +21,31 @@ export const parseQueryFilterToKQL = (filter: string, fields: Readonly<string[]>
 
   return `(${kuery})`;
 };
+
+const getPolicyQuery = (policy: string): string => {
```

Comment:
`policy` (to me) makes it sound like its the actual endpoint policy. maybe rename it to `policyId` instead?